README.md

```diff
@@ -75,10 +75,14 @@ export JAVA_HOME=`/usr/libexec/java_home -v 11`
 To install, follow these steps:
 
 ```
-$ brew install bazel
-$ brew upgrade bazel
+$ npm install -g @bazel/bazelisk
 ```
 
+This will install [bazelisk](https://github.com/bazelbuild/bazelisk) which is
```

Comment:
@subotic reading this documentation I expected `npm install -g @bazel/bazelisk` to also automatically install Bazel version 3.4.1, but that did not happen! 
<img width="869" alt="Screenshot 2020-07-20 07 26 54" src="https://user-images.githubusercontent.com/8567642/87902624-6cd53180-ca5a-11ea-8fef-a5442c8b9de8.png">